core/src/test/java/org/neo4j/ogm/metadata/schema/inheritance/Organisation.java

```diff
@@ -13,16 +13,16 @@
 
 package org.neo4j.ogm.metadata.schema.inheritance;
 
+import java.util.Set;
+
 import org.neo4j.ogm.annotation.NodeEntity;
 import org.neo4j.ogm.annotation.Relationship;
 
-import java.util.Set;
-
 /**
  * @author Frantisek Hartman
  */
 @NodeEntity
-public class Organisation extends Entity {
+public class Organisation extends Company {
```

Comment:
You forgot to commit the Company class :-)

Reply:
Also beware of the generics/associated/simple packages and which is scanned in the test...